blob/azureblob/azureblob_test.go

```diff
@@ -624,6 +624,8 @@ func TestOpenBucketFromURL(t *testing.T) {
 		{"azblob://mybucket?protocol=http", false},
 		// With invalid protocol.
 		{"azblob://mybucket?protocol=ftp", true},
+		// With Account.
+		{"azblob://mybucket?storage_account=test", false},
```

Comment:
Please also add a test to TestNewServiceURL above, similar to "Setting domain from the URL".

Reply:
added!